lib/src/json_schema/validator.dart

```diff
@@ -292,7 +292,16 @@ class Validator {
     }
 
     if (schema.contains != null) {
-      if (!instance.data.any((item) => Validator(schema.contains).validate(item))) {
+      final maxContains = schema.maxContains;
+      final minContains = schema.minContains ?? ((maxContains is int) ? 1 : null);
```

Comment:
Reading https://json-schema.org/draft/2019-09/json-schema-validation.html#rfc.section.6.4.5, should this null get turned into a value of `1` regardless of whether `maxContains` is present?

Reply:
This preserves the functionality when only `contains` is set. Earlier drafts don't have a `minContains` and it would be confusing to see an error referencing an invalid keyword.

Reply:
I guess we could check the schema's version. Would that be a code smell?